### PR TITLE
Add a description for Task 5

### PR DESCRIPTION
**Pull request by rise2semi**

### CODE REVIEW

File: task5-import-to-s3/task.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+- The task is a continuation of Homework 4 and should be done in the same repos
+- Install the latest version of AWS SDK (https://aws.amazon.com/ru/sdk-for-node-js/)
+- Install the CSV parser package (https://www.npmjs.com/package/csv-parser)
+- Sign in into AWS Console and create a new S3 bucket with a folder, called uploaded
+- Pull latest changes from nodejs-aws-fe (https://github.com/rolling-scopes-school/nodejs-aws-fe)
+
+## TASK 5.1
+---
+
+Create a new service, called **import-service**, with a its own **serverless.yml** file.
```
Comment (SergeyKovalchuk) [inline task5-import-to-s3/task.md]: At the same level as product service

your backend project structure should look like 

src/product-service
src/import-service

File: task5-import-to-s3/task.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+Create a lambda function in that **serverless.yml** file, called **importProductsFile**, which will be triggered by the HTTP **GET** method.
+
+The requested URL should be **/import**.
+
+The file name will be passed in a **query string** as a **name** parameter and should be described in the **serverless.yml** file as a **request parameter**.
```
Comment (SergeyKovalchuk) [inline task5-import-to-s3/task.md]: CSV file with products

File: task5-import-to-s3/task.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+- **3** - The **importProductsFile** lambda function returns a correct response which can be used to upload a catalog into the **S3** bucket
+- **4** - Frontend application is integrated with **importProductsFile** lambda
+- **5** - The **importFileParser** lambda function is implemented and **serverless.yml** contains configuration for the lambda
+
+## Additional (optional) tasks
+---
+
+- **+1** - **async/await** is used in lambda functions
+- **+1** - **importProductsFile** lambda is covered by **unit** tests (**aws-sdk-mock** can be used to mock S3 methods - https://www.npmjs.com/package/aws-sdk-mock)
+- **+1** - At the end of the **stream** the lambda function should move the catalog from the **uploaded** folder into the **parsed** folder
```
Comment (SergeyKovalchuk) [inline task5-import-to-s3/task.md]: What should be as a result, uploaded should be deleted? or file should be deleted? What is partial?

File: task5-import-to-s3/task.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+## Evaluation criteria
+---
+
+Reviewers should verify the lambda functions by invoking them through provided URLs.
+ 
+- **1** - File **serverless.yml** contains configuration for **importProductsFile** function
+- **3** - The **importProductsFile** lambda function returns a correct response which can be used to upload a catalog into the **S3** bucket
+- **4** - Frontend application is integrated with **importProductsFile** lambda
+- **5** - The **importFileParser** lambda function is implemented and **serverless.yml** contains configuration for the lambda
```
Comment (SergeyKovalchuk) [inline task5-import-to-s3/task.md:57]: 4 automatically close 5 point. I mean if 4 is done than 5 is done automatically

Comment (SergeyKovalchuk) [reply task5-import-to-s3/task.md:57]: We need to swap them

File: task5-import-to-s3/task.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+The file name will be passed in a **query string** as a **name** parameter and should be described in the **serverless.yml** file as a **request parameter**.
+
+Update **serverless.yml** with policies to allow lambda functions to interact with **S3**.
+
+A new **Signed URL** should be created with the following **Key: `uploaded/${fileName}`**.
+
+The response from the lambda should be the created **Signed URL**.
+
+The lambda endpoint should be integrated with the frontend by updating **import** property of the API paths configuration.
```
Comment (SergeyKovalchuk) [inline task5-import-to-s3/task.md:36]: Please provide link to FE line or module

Comment (rise2semi) [reply task5-import-to-s3/task.md:36]: Done, added path to the file

File: task5-import-to-s3/task.md (hunk 1 above)
Comment (rise2semi) [reply task5-import-to-s3/task.md]: done

File: task5-import-to-s3/task.md (hunk 2 above)
Comment (rise2semi) [reply task5-import-to-s3/task.md]: added that clarification

File: task5-import-to-s3/task.md (hunk 3 above)
Comment (rise2semi) [reply task5-import-to-s3/task.md]: Added some explanation for this task

**Pull request merged**